operator/cmd/mesh/install.go

```diff
@@ -179,6 +180,11 @@ func Install(kubeClient kube.CLIClient, rootArgs *RootArgs, iArgs *InstallArgs,
 	// Ignore the err because we don't want to show
 	// "no running Istio pods in istio-system" for the first time
 	_ = detectIstioVersionDiff(p, tag, ns, kubeClient, iop)
+	exists := revtag.PreviousInstallExists(context.Background(), kubeClient.Kube())
+	err = detectDefaultWebhookChange(p, kubeClient, iop, exists)
+	if err != nil {
+		return err
```

Comment:
This error is going to be some kube client error.
Do we want to wrap/hide it with a user friendly error message?

Reply:
Makes sense, the error could be more informative, thanks!